examples/basic/src/App.tsx

```diff
@@ -1,35 +1,27 @@
 import * as React from "react";
-import {
```

Comment:
Let's update the basic example in a separate PR.

Reply:
yeah, i did, but it slipped in here too 💀, will remove!